realsense2_camera/include/ros_node_base.h

```diff
@@ -0,0 +1,17 @@
+#ifndef ROS_NODE_BASE_HPP
```

Comment:
Please add proper copyrights (copy from other header and update the year to 2025)